### ISSUE TITLE
Nvda is not reading the description and description title labels in the office image format bar

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#10470]
### Steps to reproduce:
Open a document in word.
Now in windows explorer copy an image file and paste the file into word.
The picture will be inserted into the word document.
Now with ctrl + shift + left arrow select the inserted picture.
Press aplications, then arrow to format image, tab until Press enter.
Tab to data grid effects.
Press right arrow to stay in
Layout properties
tab to the text alt button.
Mark the button with space.
press tabe
### Actual behavior:
nvda announces edition.
Press tab and nvda announces editing again.

### Expected behavior:
nvda should list in the first tab the label of this edit box which is:
Description title.
Pressing tab again nvda should announce the title of the next edit box which is
description.
#### NVDA installed/portable/running from source:
portable
#### NVDA version:
alfa nvda.exe alpha-19051,054d7b29

#### Windows version:
10 1903 18362.449
#### Name and version of other software in use when reproducing the issue:
word WINWORD.EXE 16.0.4912.1000

#### Does the issue still occur after restarting your PC?
yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
no
log:
[alfa_alt.txt](https://github.com/nvaccess/nvda/files/3816032/alfa_alt.txt)

**Comment by Adriani90** [comment]
@fernando-jose-silva are you still having this issue in Office 2016? In Office 365 this is not reproducible.

**Comment by fernando-jose-silva** [comment]
I checked now, and didn't reproduce this issue with everything updated
thanks

**Comment by Adriani90** [comment]
Thanks for the update, I am closing as works for me.
